qiskit_optimization/algorithms/qrao/encoding_commutation_verifier.py

```diff
@@ -0,0 +1,72 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The EncodingCommutationVerifier."""
+
+from typing import Tuple
+
+from qiskit.primitives import Estimator
+
+from qiskit_optimization.exceptions import QiskitOptimizationError
+
+from .quantum_random_access_encoding import QuantumRandomAccessEncoding
+
+
+class EncodingCommutationVerifier:
+    """Class for verifying that the relaxation commutes with the objective function."""
+
+    def __init__(self, encoding: QuantumRandomAccessEncoding, estimator: Estimator = None):
+        """
+        Args:
+            encoding: The encoding to verify.
```

Comment:
Should include estimator in the args here noting it defaults to the qiskit.primitives Estimator